spec/helper.js

```diff
@@ -428,6 +428,18 @@ global.it_exclude_dbs = excluded => {
   }
 };
 
+// Fetch test exclusion list
+const testExclusionList = require('./testExclusionList.json');
+
+// Bypass test if Test UUID found in testExclusionList
+global.it_id = id => {
+  if (testExclusionList.includes(id)) {
```

Comment:
```suggestion
  if (testExclusionList?.includes(id)) {
```

Not sure but in case there is no such file, this may be needed?

Reply:
If the file doesn't exist, then require throws Module_NOT_Found so I coded it like this and it works. Is good?

```
// Fetch test exclusion list
const testExclusionList = [];
try {
  const testExclusionList = require('./testExclusionList.json');
} catch(error) {
  if(error.code === "MODULE_NOT_FOUND") {
    console.log("testExclusionList.json NOT Found")
  }
}
```

Reply:
I'd say let's only log in case a file has been found, otherwise it sounds like an error message.

Reply:
Work In Progress